rules/windows/process_creation/proc_creation_win_hktl_invoke_obfuscation_stdin.yml

```
@@ -16,22 +16,11 @@ logsource:
     category: process_creation
     product: windows
 detection:
-    selection_main:
-        # CommandLine|re: '.*cmd.{0,5}(?:\/c|\/r).+powershell.+(?:\$\{?input\}?|noexit).+\"'
+    selection:
         # Example 1: c:\windows\sYstEm32\CmD.eXE /C"echO\Invoke-Expression (New-Object Net.WebClient).DownloadString | POwersHELl -NoEXiT -"
         # Example 2: c:\WiNDOws\sysTEm32\cmd.EXe /C " ECHo Invoke-Expression (New-Object Net.WebClient).DownloadString | POwersHELl -nol ${EXEcUtIONCONTeXT}.INvOkEComMANd.InvOKEScRIPt( $InpUt )"
-        CommandLine|contains|all:
-            - 'cmd'
-            - 'powershell'
-        CommandLine|contains:
-            - '/c'
-            - '/r'
-    selection_other:
-        - CommandLine|contains: 'noexit'
-        - CommandLine|contains|all:
-              - 'input'
-              - '$'
-    condition: all of selection_*
+        CommandLine|re: 'cmd.{0,5}(?:\/c|\/r).+powershell.+(?:\$\{?input\}?|noexit).+\"'
```

```suggestion
        CommandLine|re: '(?i)cmd.{0,5}(?:\/c|\/r).+powershell.+(?:\$\{?input\}?|noexit).+\"'
```
don't know out of my head but I think `|re` should be case sensitive and even if not we should add it for consistency when looking at the other regexes

---

Yes it is case sensitive and we had a discussion around this. Hence the creation of the new modifier `i` and other specifically to be combined with regex.